bids-validator/src/schema/applyRules.ts

```diff
@@ -26,8 +26,8 @@ export function applyRules(
   if (!rootSchema) {
     rootSchema = schema
   }
-  if (!schemaPath) {
-    schemaPath = 'schema.rules'
+  if (schemaPath === undefined) {
+    schemaPath = ''
```

Comment:
Actually:

```suggestion
    schemaPath = 'rules'
```

And change L35:

```diff
-  for (const key in schema) {
+  for (const key in schema.rules) {
```